src/pages/about/_our-numbers.js

```diff
@@ -88,50 +88,50 @@ const OurNumbers = () => (
                 </article>
                 <article>
                     <Show.Desktop>
-                        <NumberHeader size="3.6rem">1.2M+</NumberHeader>
+                        <NumberHeader size="3.6rem">1.8M+</NumberHeader>
```

Comment:
please wrap these in localize function, some goes for the others 🙏 
```suggestion
                        <NumberHeader size="3.6rem">localize('1.8M+')</NumberHeader>
```